package.json

```diff
@@ -28,28 +28,28 @@
   },
   "devDependencies": {
     "@babel/cli": "7.4.4",
-    "@babel/core": "7.4.4",
+    "@babel/core": "7.4.5",
     "@babel/plugin-proposal-class-properties": "7.4.4",
     "@babel/plugin-transform-flow-comments": "7.4.4",
-    "@babel/preset-env": "7.2.0",
+    "@babel/preset-env": "7.4.5",
```

Comment:
I wonder if this version is still having the same impact as 7.4.4. @migbot can you confirm please

Reply:
Ok reverted.